File: app/src/main/java/bjzhou/coolapk/app/model/CardEntity.java
package bjzhou.coolapk.app.model;

public class CardEntity {
    private String entityType;
    private String title;
    private String url;
    private String description;
    private String pic;
    private long lastupdate;

    public String getEntityType() {
        return entityType;
    }

    public void setEntityType(String entityType) {
        this.entityType = entityType;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getUrl() {
        return url;
    }

    public void setUrl(String url) {
        this.url = url;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public String getPic() {
        return pic;
    }

    public void setPic(String pic) {
        this.pic = pic;
    }

    public long getLastupdate() {
        return lastupdate;
    }

    public void setLastupdate(long lastupdate) {
        this.lastupdate = lastupdate;
    }

    @Override
    public String toString() {
        return "CardEntity{" +
                "entityType='" + entityType + '\'' +
                ", title='" + title + '\'' +
                ", url='" + url + '\'' +
                ", description='" + description + '\'' +
                ", pic='" + pic + '\'' +
                ", lastupdate=" + lastupdate +
                '}';
    }
}

File: app/src/main/kotlin/bjzhou/coolapk/app/model/CardEntity.kt
package bjzhou.coolapk.app.model

class CardEntity {
    var entityType: String? = null
    var title: String? = null
    var url: String? = null
    var description: String? = null
    var pic: String = ""
    var lastupdate: Long = 0

    override fun toString(): String {
        return "CardEntity{" +
                "entityType='" + entityType + '\'' +
                ", title='" + title + '\'' +
                ", url='" + url + '\'' +
                ", description='" + description + '\'' +
                ", pic='" + pic + '\'' +
                ", lastupdate=" + lastupdate +
                '}'
    }
}
